assets/site/guide.sass

```diff
@@ -142,15 +142,13 @@ html.cockpit-guide
 
   h2, h3, h4, h5, h6,
   dl.toc .part,
-  dl.toc .chapter,
```

Comment:
This is the change that you describe in the commit message I guess.

Reply:
Yes this the one.